app/models/product.rb

```diff
@@ -334,6 +334,10 @@ def alert
     nil
   end
 
+  def duration_pricing_mode?
+    false
+  end
```

Comment:
Given you added this, can you please update all occurrences of `@product.is_a?(Instrument) && @product.duration_pricing_mode?` to be `@product.duration_pricing_mode?`?